src/components/SQForm/SQFormAutocomplete.js

```diff
@@ -162,12 +165,12 @@ function SQFormAutocomplete({
         disableListWrap
         classes={classes}
         ListboxComponent={ListboxVirtualizedComponent}
-        options={children}
+        options={[...children, EMPTY_OPTION]}
         onBlur={handleAutocompleteBlur}
         onChange={handleAutocompleteChange}
         onInputChange={handleInputChange}
         inputValue={inputValue}
-        value={initialValue}
+        value={initialValue || EMPTY_OPTION}
```

Comment:
[question] - If `displayEmpty` is `false` how come we are adding the empty option to the dropdown options on line 168? Same question on line 173 about falling back to an empty option.

I only expect to see the empty dropdown option when `displayEmpty` is `true`

(I haven't had a chance to watch the loom yet 😬)

Reply:
Whoops, I meant to remove `displayEmpty` entirely. This empty option is required to avoid to the uncontrolled/controlled warning beacuse of the way autocomplete works. It *needs* an option for whatever value is provided. If we provide an empty string there *has* to be an option there. So I decided to always include that option.

Validation wise, if this field is required the user will be unable to select this empty option and proceed as an empty string, the value of this empty option, will still fail the validation.